[Moodle] Add "supportedPhpVersions" field (#3981)
captn3m0/endoflife.date: products/moodle.md

From Moodle 2.6 onwards, the end of support, both general and security, happens the second Monday
of May and November, observing the 12, 18... month periods, no matter if the major release was
delayed or not.

## PHP Support

> {% include table.html
> labels="Release,Supported PHP versions"
> fields="releaseCycle,supportedPhpVersions"
> types="string,string"
> rows=page.releases %}

Reviewer: ```suggestion
{%- assign collapsedCycles = page.releases | collapse_cycles:"supportedJavaVersions"," - " %}
{% include table.html
labels="Release,Supported PHP versions"
fields="releaseCycle,supportedPhpVersions"
types="string,string"
rows=collapsedCycles %}
```

For a more condensed table.

> See [details](https://moodledev.io/general/development/policies/php) and check exceptions.

Reviewer: ```suggestion
More details are available on [Moodle's PHP page](https://moodledev.io/general/development/policies/php).
```

From Moodle 2.6 onwards, the end of support, both general and security, happens the second Monday
of May and November, observing the 12, 18... month periods, no matter if the major release was
delayed or not.

## PHP Support

> {%- assign collapsedCycles = page.releases | collapse_cycles:"supportedJavaVersions"," - " %}

Reviewer: ```suggestion
{%- assign collapsedCycles = page.releases | collapse_cycles:"supportedPhpVersions"," - " %}
```

{% include table.html
labels="Release,Supported PHP versions"
fields="releaseCycle,supportedPhpVersions"
types="string,string"
rows=collapsedCycles %}

> More details are available on [Moodle's PHP page](https://moodledev.io/general/development/policies/php).

Reviewer: ```suggestion
For details about PHP support, see [Moodle's PHP page](https://moodledev.io/general/development/policies/php).
```
